GarbleExecutable.py

```diff
@@ -0,0 +1,91 @@
+import garble
+import wx
+from multiprocessing import freeze_support
+import sys
+import os
+from pathlib import Path
+
+#pyinstaller GarbleExecutable.py  --onefile -w --add-data ./venv/Lib/site-packages/clkhash/data;clkhash/data --add-data ./venv/Lib/site-packages/clkhash/schemas;clkhash/schemas --add-data ./example-schema;example-schema --add-data ./secret-file/secret-file.txt;secret-file
+
+
+if getattr(sys, 'frozen', False) and hasattr(sys, '_MEIPASS'):
+    os.chdir(sys._MEIPASS)
+
+
+class CSVManager(wx.Frame):
+    def __init__(self, *args, **kwargs):
+        super(CSVManager, self).__init__(*args, **kwargs)
+        self.PII_path = ""
+        self.output_dir = ""
+        self.schema_dir = "example-schema"
+        self.salt_path = "secret-file/secret-file.txt"
+        self.txt1 = None
```

Comment:
Field names should be more descriptive than `txt1`. This could be `self.csv_file_location`